sql/database.sql

```diff
@@ -2527,7 +2527,7 @@ INSERT INTO `gacl_aro_sections_seq` VALUES (9);
 
 DROP TABLE IF EXISTS `gacl_aro_seq`;
 CREATE TABLE `gacl_aro_seq` (
-  `id` int(11) NOT NULL
+  `id` int(12) NOT NULL
```

Comment:
Is it necessary to come up with a sql statement to ensure the value of this table is at least 12 in the upgrade script?

Reply:
This number ends up being meaningless for an int :)
the allowable range for an int is -2147483648 to 2147483647
That number for an int is only used for "minimal display width":
https://stackoverflow.com/questions/4055564/what-does-the-number-in-parenthesis-really-mean

Reply:
I think that the gacl_aro_seq will need to be incremented at the upgrade script and that official_additional_users.sql script.

Reply:
prob something like (assuming the value in gacl_aro_seq is already being used; it may be that the value there is the next one to use)
get the gacl_aro_seq + 1
replace gacl_aro_seq with this new value
use this value in the gacl_aro insert

replace gacl_aro_seq with this new value

Reply:
totally forgot about that discussion we had regarding it on a sat call last year